batavia/VirtualMachine.js

```diff
@@ -1300,15 +1300,26 @@ VirtualMachine.prototype.byte_COMPARE_OP = function(opnum) {
 VirtualMachine.prototype.byte_LOAD_ATTR = function(attr) {
     var obj = this.pop();
     var val;
-    if (obj.__getattr__ === undefined) {
+
+    if (obj.__getattribute__ === undefined) {
```

Comment:
This appears to be wrong -- we should be checking for `__getattr__` and `__getattribute__` on the class, not on the instance:
```python
>>> class A:
...     pass
... 
>>> a = A()
>>> a.__getattr__ = lambda x:x
>>> 
>>> a.asdasdsad
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'A' object has no attribute 'asdasdsad'
>>> A.__getattr__ = lambda instance,x:x
>>> a.asdasdsad
'asdasdsad'
```